layouts/index.html

```diff
@@ -27,11 +27,11 @@ <h1 class="usa-display">{{ $translation.homepage.header }}</h1>
             </option>
             {{ if eq $.Site.Params.language "english" }}
               {{ range where .Data.Pages.ByTitle "Section" "register" }}
-                <option value="{{ replace .Title "." "" | lower | urlize }}">{{ title .Title }}</option>
+                <option value="{{ .Params.state_abbreviation | lower }}">{{ title .Title }}</option>
```

Comment:
Alright, I think I understand. Aren't the abbreviations the same for both languages? Seems like we can clean up the key to simply be `register` so we can use it for both languages?

```
            {{ if eq $.Site.Params.language "english" }}
              {{ range where .Data.Pages.ByTitle "Section" "register" }}
                <option value="{{ .Params.state_abbreviation | lower }}">{{ title .Title }}</option>
              {{ end }}
            {{ else }}
              {{ range where .Data.Pages.ByTitle "Section" "registrar" }}
                <option value="{{ .Params.state_abbreviation | lower }}">{{ title .Title }}</option>
              {{ end }}
            {{ end }}
```

becomes 

```
              {{ range where .Data.Pages.ByTitle "Section" "register" }}
                <option value="{{ .Params.state_abbreviation | lower }}">{{ title .Title }}</option>
              {{ end }}
```